src/output.c

```diff
@@ -239,6 +246,34 @@ static void output_id3(output_t *st, unsigned int program, uint8_t *buf, unsigne
                           price, until, url, seller, desc, received_as);
             }
         }
+        else if (memcmp(tag, "COMM", 4) == 0)
+        {
+            if (frame_len < 5)
+            {
+                log_warn("bad COMM tag (frame_len %d)", frame_len);
+            }
+            else
+            {
+                uint8_t *delim = memchr(data+4, 0, frame_len);
```

Comment:
I don't think this will work for UCS2, where the terminator is `00 00`.

Also, it appears this could read past the end of the buffer; `frame_len` should be `frame_len - 4`.